core/vm/opcodes.go

```diff
@@ -198,9 +198,16 @@ const (
 	LOG4
 )
 
+// 0xb0 range - subroutine ops.
+const (
+	JUMPSUB   OpCode = 0xb3
+	BEGINSUB  OpCode = 0xb5
+	RETURNSUB OpCode = 0xb7
+)
```

Comment:
please clarify the values not matching the spec, cc https://github.com/openethereum/openethereum/pull/11612#discussion_r404788297